### PR TITLE
feat: add uan for token list

### PR DESCRIPTION
**Pull request by zhangyouxin**
This PR enhances:

- add UAN for testnet v0/testnet v1/mainnet v0/mainnet v1 token lists

referrence: https://github.com/nervosnetwork/rfcs/pull/335

### CODE REVIEW

**Comment by zhangyouxin** [comment]
image

image

**Comment by zhangyouxin** [comment]
Changed to:

 - show token symbol in currency panel
 - hover over the token and UAN will show up as tooltip

---

image

File: src/light-godwoken/lightGodwokenType.ts [hunk 1]
```diff
@@ -15,15 +15,18 @@ export interface Token {
   decimals: number;
   tokenURI: string;
 }
+export interface TokenExtra extends Token {
+  hover?: string;
+}
```
Comment (homura) [inline src/light-godwoken/lightGodwokenType.ts:21]:
```suggestion
export interface UniversalToken extends Token {
  /**
  * Universal Asset Notation
  * @see https://github.com/nervosnetwork/rfcs/blob/a092218b8f3ba9b6616ff41bd56a5a75d42efaf7/rfcs/0000-universal-asset-notation/0000-universal-asset-notation.md
  **/
  uan: string;
}
```

**Review by homura — APPROVED** [review]

**Comment by homura** [comment]
maybe we can replace `${token.symbol} ${token.route}` with `${token.name} ${token.route}` in `displayName` to make the `displayName` more readable

**Pull request merged**